spring-cloud-starter-stream-processor-tensorflow/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-starter-stream-processor-tensorflow</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-starter-stream-processor-tensorflow</name>
+	<description>Spring Cloud Stream processor-tensorflow core</description>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>tensorflow-app-starters-build</artifactId>
+		<version>1.2.1.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.tensorflow</groupId>
+			<artifactId>tensorflow</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-bus</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-config</artifactId>
```

Comment:
Also part of the model reconfiguration (https://github.com/tzolov/tensorflow-spring-cloud-stream-app-starters/issues/2).
The idea originates from @markfisher and @sabbyanandan : "config server could possibly host the Resource URI, and the property value that is annotated with `@RefreshScope` can be of type `Resource` - the bean into which its injected will receive the updated value (of course that requires that the URI would change to point to a new actual resource when the model is updated)"